ros_ign_gazebo_demos/worlds/vehicle.sdf

```diff
@@ -0,0 +1,106 @@
+<?xml version="1.0" ?>
+<!--
+  Ignition Gazebo sdformat_urdf parser plugin demo
+
+  Try sending commands:
+
+    ign topic -t "/model/vehicle/cmd_vel" -m ignition.msgs.Twist -p "linear: {x: 1.0}, angular: {z: -0.1}"
+
+  Listen to odometry:
+
+    ign topic -e -t /model/vehicle/odometry
+
+-->
+<sdf version="1.8">
+  <world name="demo">
+    <physics name="1ms" type="ignored">
+      <real_time_factor>0</real_time_factor>
```

Comment:
For this kind of demo, I think it's more user-friendly to run at real time, i.e. `1.0`. That's already the default, so we can remove the entire `physics` block.